src/widgets/ASlider.cpp

```diff
@@ -530,6 +530,7 @@ LWSlider::LWSlider(wxWindow *parent,
       speed = 0.5;
       break;
    case FRAC_SLIDER:
+   case PERCENT_SLIDER:
```

Comment:
The commit that adds PERCENT_SLIDER for the play and record meters is perhaps incomplete.

I can double-click on the slider thumb and get a dialog to enter a value.  But it must be a fraction from 0 to 1, not a percentage value.  Is that acceptable?  A design question @Tantacrul

Reply:
I also see that not all calls to SendUpdate() are followed by ShowTip()  -- the dialog has some of those calls.  Should ShowTip() be added?

Reply:
At this commit, at least in macOS, I do not yet get the hover texts over these two slider thumbs.  But I do get them for track pan and gain and the mixer board gain slider.  Maybe a later commit will fix this. I haven't built them all yet.